src/client/components/MusicReserve/TrackTable.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { CondensedButton } from '@instructure/ui-buttons';
+import { Table } from '@instructure/ui-table';
+
+export const TrackTable = ({ album, handleClick }) => {
+  TrackTable.propTypes = {
+    album: PropTypes.object,
+    handleClick: PropTypes.func,
+  };
+
+  return (
+    <Table caption="Tracks">
+      <Table.Head>
+        <Table.Row>
+          <Table.ColHeader id="title">Track title</Table.ColHeader>
+          <Table.ColHeader id="album">Album</Table.ColHeader>
+        </Table.Row>
+      </Table.Head>
+      <Table.Body>
+        {album.items.map(track => (
+          <Table.Row key={track.trackTitle}>
```

Comment:
@maxpluez I am not sure if it is good practice to put the full title as a an id.. But I seem in the data webservice that for albums that have more than one "track" that there is usually "trackNumber" set. Can you use that instead?

Reply:
@maxpluez Well, I see some with tracknumber empty.. What about combining volume-disc-side-tracknumber?